predict-olympic-medals/BaselineSolution.ipynb

```diff
@@ -334,9 +342,9 @@
     }
    ],
    "source": [
-    "num_medals = ft.Feature(es['medals_won']['medal_id'], parent_entity=es['countries_at_olympic_games'], primitive=Count)\n",
-    "mean_num_medals = ft.Feature(num_medals, parent_entity=es['countries'], primitive=Mean).rename(\"mean_num_medals\")\n",
-    "mean_num_medals.entity.id"
+    "num_medals = ft.Feature(es['medals_won'].ww['medal_id'], parent_dataframe_name='countries_at_olympic_games', primitive=Count)\n",
+    "mean_num_medals = ft.Feature(num_medals, parent_dataframe_name='countries', primitive=Mean).rename(\"mean_num_medals\")\n",
```

Comment:
these lines are kind of long--maybe worth splitting up into multiple

Reply:
Updated!